scriptworker/constants.py

```diff
@@ -176,14 +176,16 @@
     # Map scopes to restricted-level
     'cot_restricted_scopes': frozendict({
         'firefox': frozendict({
-            'project:releng:balrog:release': 'all-release-branches',
-            'project:releng:beetmover:release': 'all-release-branches',
+            'project:releng:balrog:server:release': 'all-release-branches',
```

Comment:
* I haven't added esr yet.  We can when we uplift to esr52, after we've proven this model.

* You mean aurora is in `all-release-branches`? Yeah, we can remove it if we're planning on doing all release branches on jamun or something.